source/Irrlicht/COpenGLDriver.cpp

```diff
@@ -71,7 +71,7 @@ namespace video
 COpenGLDriver::COpenGLDriver(const irr::SIrrlichtCreationParameters& params,
 		io::IFileSystem* io, CIrrDeviceWin32* device, const asset::IGLSLCompiler* glslcomp)
 : CNullDriver(device, io, params.WindowSize), COpenGLExtensionHandler(),
-	runningInRenderDoc(false),  CurrentRenderMode(ERM_NONE), ResetRenderStates(true), ColorFormat(asset::EF_R8G8B8_UNORM), Params(params),
+	runningInRenderDoc(false),  /*ResetRenderStates(true),*/ ColorFormat(asset::EF_R8G8B8_UNORM), Params(params),
```

Comment:
I think resetRenderStates used to mean, flush state to default first chance you get

Basically ResetRenderStates=true would be the equivalent of setting the next graphics pipeline state to its default values (although thats crazy inefficient and pointless most of the time)